### PR TITLE
EnterDAO && KingDefi && Sorbet Finance TVL Adapters

### PR DESCRIPTION
**Pull request by Mohammed18062020**
##### Twitter Link:
EnterDAO -> https://twitter.com/EnterDao
KingDefi -> https://twitter.com/KingDefi2
Gelato Network (Sorbet Finance) -> https://twitter.com/gelatonetwork

##### Website Link:
EnterDAO -> https://enterdao.xyz/
KingDefi -> https://kingdefi.io/king-farms
Gelato Network (Sorbet Finance) ->https://www.gelato.network/

##### Logo (High resolution, preferably in .svg and .png, for application on both white and black backgrounds. Will be shown with rounded borders):
EnterDAO -> 
image

KingDefi -> 
image

Gelato Network (Sorbet Finance)  -> 
image

##### Current TVL:
EnterDAO -> $98,919,275
KingDefi -> $5,050,160$
Gelato Network (Sorbet Finance) -> $32.45 M

##### Chain:
EnterDAO -> Ethereum
KingDefi -> BSC
Gelato Network (Sorbet Finance)-> 32.45 M

##### Coingecko ID (so your TVL can appear on Coingecko): (https://api.coingecko.com/api/v3/coins/list)
EnterDAO -> "id": "enterdao"
KingDefi -> 
Gelato Network (Sorbet Finance) -> "id": "gelato"

##### Coinmarketcap ID (so your TVL can appear on Coinmarketcap): (https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)
EnterDAO -> "id":12228
KingDefi -> "id":11024
Gelato Network (Sorbet Finance) -> "id":7229

##### Token address and ticker if any:
EnterDAO Token (ENTER) -> 0xd779eEA9936B4e323cDdff2529eb6F13d0A4d66e
KingDefi Token (KRW) -> 0x1446f3CEdf4d86a9399E49f7937766E6De2A3AAB
Gelato Token (GEL) -> 0x15b7c0c907e4C6b9AdaAaabC300C08991D6CEA05

##### Category (Yield/Dexes/Lending/Minting/Assets/Insurance/Options/Indexes/Staking) *Please choose only one:
EnterDAO -> Staking
KingDefi -> Yield
Gelato Network (Sorbet Finance) -> Staking

### CODE REVIEW

**Comment by Mohammed18062020** [comment]
Sorbet Finace is a product of Gelato Network.

File: projects/enterdao/index.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+const USDC_ENTR_SUSHI_LP = "0x83b546e10917432a722444672504f0d459472171";
+const yieldFarms = [
+  "0x6b3595068778dd592e39a122f4f5a5cf09c90fe2", //SUSHI
+  "0xbb0e17ef65f82ab018d8edd776e8dd940327b28b", //AXS
+  "0x0391D2021f89DC339F60Fff84546EA23E337750f", //BOND
+  "0x0f5d2fb29fb7d3cfee444a200298f468908cc942", //MANA
+  "0x618679df9efcd19694bb1daa8d00718eacfa2883", //XYZ
+  "0x767fe9edc9e0df98e07454847909b5e959d7ca0e", //ILV
+  "0x3845badAde8e6dFF049820680d1F14bD3903a5d0", //SAND
+  "0x83b546e10917432a722444672504f0d459472171", //SUSHI-LP
```
Comment (0xngmi) [inline projects/enterdao/index.js:14]: is this pool2?

Comment (0xngmi) [reply projects/enterdao/index.js:14]: seems so, would be best to separate

File: projects/kingdefi/index.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  const balances = {};
+
+  let transformAddress = await transformBscAddress();
+
+  await addFundsInMasterChef(
+    balances,
+    MasterChefContract,
+    chainBlocks["bsc"],
+    "bsc",
+    transformAddress,
```
Comment (0xngmi) [inline projects/kingdefi/index.js:20]: need to add the ignored token

**Comment by 0xngmi** [comment]
will apply the fixes myself

**Comment by Mohammed18062020** [comment]
Ok sorry,, I've just seen the comments!

**Pull request merged**